sql_datafreshness/sql_datafreshness.py

```diff
@@ -56,6 +56,9 @@ def get_args():
     parser.add_argument('--conf', dest='config_file',
                         help='Configuration file',
                         required=False)
+    parser.add_argument('--send', dest='send',
+                        help='Send the information to the mail (true or false)',
```

Comment:
* Isn't it third person? It should be "Sends" then. 
* Don't expect to receive a "true" or "false" through the command line, have a look at how to use boolean parameters